libexternal/src/osxsupport.cpp

```diff
@@ -39,7 +39,7 @@ char *string_to_utf8(const char *p_string)
 	if (s_text_to_unicode_info != NULL)
 	{
 		unsigned int t_string_length;
-		t_string_length = strlen(p_string);
+		t_string_length = (unsigned int)strlen(p_string);
```

Comment:
Hmm... what are you're thoughts on this which will truncate the string to the maximum size we can fit into our unfortunately guessed buffer size:
```
size_t t_string_length;
t_string_length = strlen(p_string);
if (t_string_length > SSIZE_MAX)
            t_string_length = SSIZE_MAX;
```